### PR TITLE
Buzzoola bid adapter

### PR DESCRIPTION
**Pull request by VanCyric**
## Type of change
- [x] New bidder adapter 

## Description of change
Added new Adapter for [Buzzoola](https://buzzoola.com)

- code: modules/buzzoolaBidAdapter.js
- doc: modules/buzzoolaBidAdapter.md
- unit test: test/spec/modules/buzzoolaBidAdapter_spec.js

- test parameters for validating bids
```
{
  bidder: 'buzzoola',
  params: {
    placementId: 417846
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer: <devteam@buzzoola.com>
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/: <https://github.com/prebid/prebid.github.io/pull/1456>

## Other information

### CODE REVIEW

**Comment by Fawke** [comment]
Hi @VanCyric,

Thanks for the PR. There's one problem, when I tried to plug in your adapter with the test parameters you've shared in the [hello world](https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/hello_world.html) example, I got this error. 

Screenshot 2019-09-05 at 12 30 20 PM

Can you look into it? Thanks.

**Comment by VanCyric** [comment]
Hi, @Fawke.
Thanks for noticing this error. I've added some code to handle empty responses properly.
Did you use ad unit sizes specified in the description of the adapter? I was not sure if I should include the sizes in PR description because they are not a part of adapter-level settings.

File: modules/buzzoolaBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            params: {
+                placementId: 417846
+            }
+        }]
+    },
+    // Video instream adUnit
+    {
+        code: 'video-instream',
+        mediaTypes: {
+            video: {
```
Comment (Fawke) [inline modules/buzzoolaBidAdapter.md]: I think you forgot the add the context property to the video object.

Comment (VanCyric) [reply modules/buzzoolaBidAdapter.md]: added

**Pull request merged**